### PR TITLE
Feature/nex 81 es2015 migration

### PR DESCRIPTION
**Pull request by krampstudio**
Related to https://oat-sa.atlassian.net/browse/NEX-81

 - Bundles with babel
 - migrate some controllers to ES2015 (partially)

Requires PR : 

 - [ ] https://github.com/oat-sa/tao-core/pull/2156

### CODE REVIEW

File: views/build/grunt/bundle.js [hunk 1]
```diff
@@ -19,6 +19,8 @@
 /**
```
Comment (ferenckiss89) [inline views/build/grunt/bundle.js]: update the copyright dates pls

File: views/js/controller/viewResult.js [hunk 2]
```diff
@@ -44,16 +61,17 @@ define([
...
+                        filterSubmission: $resultFilterField.select2('val'),
+                        filterTypes: classFilter
+                    }
+                );
             });
 
 
             //bind the download buttons
-            $('.download', $container).on("click", function (e) {
+            $('.download', $container).on("click", function () {
```
Comment (ferenckiss89) [inline views/js/controller/viewResult.js]: arrow fn and single quotation?

File: views/js/controller/viewResult.js [hunk 3]
```diff
@@ -65,13 +83,13 @@ define([
             });
 
             $('.preview', $container).on("click", function (e) {
```
Comment (ferenckiss89) [inline views/js/controller/viewResult.js]: arrow fn and single quotation?

File: views/js/controller/viewResult.js [hunk 4]
```diff
@@ -15,27 +32,27 @@ define([
...
             }).select2('val', conf.filterSubmission || 'all');
 
-            for(var i in classFilter){
-                $('[value="'+classFilter[i]+'"]').prop('checked', 'checked');
+            for(let i in classFilter){
+                $(`[value="${classFilter[i]}"]`).prop('checked', 'checked');
             }
 
-            $('.result-filter-btn', $container).click(function(e) {
+            $('.result-filter-btn', $container).click(function() {
```
Comment (anshul119) [inline views/js/controller/viewResult.js]: can be arrow func

File: views/js/controller/viewResult.js (hunk 3 above)
Comment (krampstudio) [reply views/js/controller/viewResult.js]: I've updated the quote but let the function since the lexical scope is used within it

**Pull request merged**